packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -1,20 +1,10 @@
-import type {Modal} from '../types';
-
 export interface NavigationApiContent {
-  /** Contains the information about the currently presented modal */
-  currentModal: Modal;
-
-  /** Presents a new modal on top of the current one.
-   * @param modal this interface contains the information about the path and any initial parameters.
-   */
-  presentModal(modal: Modal): void;
```

Comment:
We no longer need this. The `navigate` function will go to the correct modal if you pass in the modal's name. This matches more closely what react-navigation does.